style/stylesheets/betterttv-dark.css

```diff
@@ -2156,3 +2156,32 @@ span.subscribe-text {
 .manager .meta .actions {
     border-color: #676767;
 }
+
+/* creative tag filter arrows */
+.creativetag-list-contain .filter-nav {
+    background: #333;
+}
+
+.creativetag-list-contain .filter-nav:before {
+    margin-top: -9px;
+    border: 5px solid #fff;
+    border-top-color: transparent;
+    border-bottom-color: transparent;
+    opacity: .5;
+}
+
+.filter-nav.goleft {
```

Comment:
This is what Twitch is using their stylesheet.
![Twitch's stylesheet](https://i.gyazo.com/86644fa4bb059de17e8610e8f434b802.png)